src/app/view/components/dot-workflow-task-detail/dot-workflow-task-detail.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input, EventEmitter, Output, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { DotWorkflowTaskDetailService } from './services/dot-workflow-task-detail.service';
+
+/**
+ * Allow user to view a workflow task to a DotCMS instance
+ *
+ * @export
+ * @class DotWorkflowTaskDetailComponent
+ * @implements {OnInit}
+ */
+@Component({
+    selector: 'dot-workflow-task-detail',
+    templateUrl: './dot-workflow-task-detail.component.html'
+})
+export class DotWorkflowTaskDetailComponent implements OnInit {
+    @Output() close: EventEmitter<any> = new EventEmitter();
+    url: Observable<string>;
```

Comment:
By styleguide should be `url$` because is an Observable (I change a couple in my end)